### DISCUSSION TITLE
Some information on the UI?

### DISCUSSION THREAD

**Discussion by gomaxlou** [comfyanonymous/ComfyUI#762] [Q&A]
![image](https://github.com/comfyanonymous/ComfyUI/assets/37968309/dab3b38d-a50d-4d9c-a0ca-38bb3d1ef221)
What do the representatives mean?

**Comment by reaper47** [comment]
T is the graph's global time, most likely the elapsed time.
I is the number of iterations.
N is the number of nodes in the graph. The number within the square brackets is the number of visible nodes in your view port.
V is the graph's version.
FPS is the graph's frames per second.

**Reply by gomaxlou** [reply]
but the T , I always 0 like screenshot?

**Reply by reaper47** [reply]
It is indeed always 0 for some reason. Maybe there is a way to activate those values.
